packages/sdk/tsdx.config.js

```diff
@@ -0,0 +1,20 @@
+const replace = require('@rollup/plugin-replace');
+const envType = process.env.ENV_TYPE;
+
+module.exports = {
+  rollup(config) {
+    // removes the esm prefix -> that just makes importing the module more difficult
+    config.output.file = config.output.file.replace('.esm', '');
+
+    if (envType == 'web')
+      // unshift so that this plugin gets run first
+      config.plugins.unshift(
+        replace({ 
+          '@uma/contracts-node': "@uma/contracts-frontend",
```

Comment:
This is cool. Just to double check, is a string replace the standard way to replace imports? Also, does this replace it in the compiled output or does it modify the ts _before_ compilation? I assume the latter right?

Reply:
i didnt find any sources to show how to change imports based on environment during the build process. I kind of just tried this based on a slightly different tutorial for changing out code based on environment.  So no i dont think this is standard, but was not able to find a standard way to do this. 

I specifically had to push this plugin to the front of the stack so it ran before build. If you push this to the back the plugin array the build will fail!